frontend/src/components/Course/Course.tsx

```diff
@@ -13,25 +16,66 @@ import {
   ButtonsContainer,
   CourseDescriptionWrapper,
   CourseInfoBox,
+  MobileCourseInfoBox,
+  MobileCourseProgress,
+  MobileCourseCompleted,
+  CourseTextContainer,
 } from './styled';
 
 interface Props {
   title: string | undefined;
   description: string;
   duration: string | undefined;
   lessons: number | undefined;
+  pageName?: string;
+  status?: string;
+  progress?: number;
 }
 
-const CourseItem: React.FC<Props> = ({ title, description, duration, lessons, children }) => (
+const PAGES = {
+  myCourses: 'myCourses',
+  coursesList: 'coursesList',
+};
+
+const CourseItem: React.FC<Props> = ({
+  title,
+  description,
+  duration,
+  lessons,
+  pageName,
+  status,
+  progress,
+  children,
+}) => (
   <CourseContainer container direction="column">
     <AboutCourseContainer>
       <ImageWrapper>
         <Image />
       </ImageWrapper>
-      <CourseTitle>{title}</CourseTitle>
-      <CourseDescriptionWrapper>
-        <CourseDescription>{shortifyCourseDescription(description)}</CourseDescription>
-      </CourseDescriptionWrapper>
+      <CourseTextContainer>
+        <CourseTitle>{title}</CourseTitle>
+        <CourseDescriptionWrapper>
+          <CourseDescription>{shortifyCourseDescription(description)}</CourseDescription>
+        </CourseDescriptionWrapper>
+        <MobileCourseInfoBox>
+          <CourseInfo duration={duration} lessons={lessons} />
+        </MobileCourseInfoBox>
+        {status === STATUS.completed && (
+          <MobileCourseCompleted>
+            Completed <img alt="" src={checkIcon} />
+          </MobileCourseCompleted>
+        )}
+      </CourseTextContainer>
+      {pageName === PAGES.myCourses && status !== STATUS.completed && (
+        <MobileCourseProgress>
+          <ProgressBar
+            value={progress}
+            text={`${progress}%`}
+            size="medium"
+            variant="mobileCourse"
+          />
+        </MobileCourseProgress>
+      )}
     </AboutCourseContainer>
     <ButtonsContainer>
```

Comment:
We have problem with position button block  like on this screen (https://radikal.ru/lfp/b.radikal.ru/b23/2202/a4/b5b1e3854eef.png/htm)
https://a.radikal.ru/a29/2202/93/1f982dbf23ef.png